content/docs/install/helm-install/index.md

```diff
@@ -6,63 +6,123 @@ weight: 40
 
 Keptn is installed using a Helm chart using the Helm CLI.
 You must install the [Helm CLI](https://helm.sh)
-and the [Keptn CLI](../cli-install)
 before attempting to install Keptn.
 
+You probably want to also install the [Keptn CLI](../cli-install)
+before installing Keptn.
+It is possible to do most of what you need to do on modern releases of Keptn without the Keptn CLI
+but the CLI provides additional functionality that is useful.
 After installing Keptn,
 you must [authenticate the Keptn CLI](../authenticate-cli-bridge/#authenticate-keptn-cli). 
 
 Keptn consists of a **Control Plane** and an **Execution Plane**.
 
-* The **Control Plane** allows using Keptn for the [Quality Gates](../../concepts/quality_gates/)
-  and [Automated Operations](../../concepts/automated_operations/).
+* The **Control Plane** is the minimum set of components that are required
+  to run a Keptn instance  and to manage projects, stages, and services;
+  to handle events; and to provide integration points.
+  The control plane orchestrates the task sequences defined in Shipyard
+  but does not actively execute the tasks.
   The Keptn Control Plane can run with no Execution Plane configured.
-* The **Control Plane with the Execution Plane** is required to implement
-  [Continuous Delivery](../../concepts/delivery/)
-  on top of Quality Gates and Automated Operations..
-  * Note that you must install either the
-  [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)
-  or [Istio](https://istio.io) on the Kubernetes cluster(s) where the Execution Plane is installed.
+
+  The Control Plane can implement the [Quality Gates](../../concepts/quality_gates/)
+  and [Automated Operations](../../concepts/automated_operations/) features
+  but does not run microservices that integrate other tools with Keptn.
+
+* The **Execution Plane** runs the microservices that integrate other tools with Keptn
+  and is installed when you install the first microservice on the cluster.
+  For example, the JMeter test tool runs on the Execution Plane
+  and installing it installs the Execution Plane.
+  As another example, the [Continuous Delivery](../../concepts/delivery/) feature
+  requires that you install either the
+[Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)
+or [Istio](https://istio.io) on the Kubernetes cluster(s)
+and they execute on the Execution Plane.
```

Comment:
I don't think this paragraph really makes sense. The concept is a little different compared to how it's described here.
The execution plane **is** the set of keptn integrations that the user installs, there is nothing more to it and nothing else will be installed alongside the user-installed integrations.

Maybe @thisthat has some more input on better wording here?

Reply:
As @mowies said, the "execution plane" really jsut means "not the same namespace as the core Keptn microservices". So the "execution plane" isn't really a thing, it's just denoting "somewhere else". Imagine I install Keptn in on k8s cluster 1. I then install **only** the job executor integration (or any other integration) on cluster 2. Important to note that cluster 2 is a plain vanilla k8s cluster. The act of installing job executor means I can now describe cluster 2 as my "execution plane".

Another example. Cluster 1 with keptn core installed. I create a non-containerised shell script that polls the Keptn API every 5 seconds. I run the shell script on my laptop. Technically my laptop is now the "execution plane"/

Reply:
FWIW the "control plane" also isn't a "thing". It's a logical construct meaning "the place I've installed the set of core Keptn microservices".

Reply:
Thanks so much for this!  I think the existing docs really make it sound like the control plane and execution plane are Things but, as I've been working through various pieces, I was starting to get the idea that this is not the case.  So now I know...

Reply:
Okay, I think this paragraph may be straightened out.  See what you think